lib/tools/adb-emu-commands.js

```diff
@@ -43,4 +65,45 @@ emuMethods.sendSMS = async function (phoneNumber, message = '') {
   await this.adbExecEmu(['sms', 'send', phoneNumber, message]);
 };
 
+emuMethods.gsmCall = async function (phoneNumber, action = '') {
+  if (GSM_CALL_ACTIONS.indexOf(action) === -1) {
+    log.errorAndThrow(`Invalid gsm action param ${action}`);
```

Comment:
it might be useful to show the list of supported values in these error messages. This will help the client to put the correct value.